### PR TITLE
Made firewalld rules more specific for RHEL

### PR DESCRIPTION
**Pull request by my0373**
Added specific firewall rules for RHEL and made sure firewalld is running.

This should fix task #101 and #117

Tested by spinning up a lab and testing ping from the control node.

### CODE REVIEW

**Comment by my0373** [comment]
Hi, is it ok to review this ? Thanks!

File: tools/aws_lab_setup/roles/common/tasks/RedHat.yml [hunk 1]
```diff
@@ -12,9 +12,22 @@
...
+- name: RHEL | Enable specific firewall ports
+  firewalld:
+    immediate: yes
+    permanent: yes
+    service: "{{ item }}"
+    state: enabled
+  with_items: "{{ firewall_ports }}"
+  notify: restart firewalld
+
+- name: RHEL | Start firewalld
```
Comment (samdoran) [inline tools/aws_lab_setup/roles/common/tasks/RedHat.yml]: `firewalld` needs to be running _before_ trying to enable ports or it will fail.

Comment (my0373) [inline tools/aws_lab_setup/roles/common/tasks/RedHat.yml]: You are right, apologies I thought i'd corrected that. 

Will fix it and issue a new PR. Thanks

**Comment by my0373** [comment]
You'll have to excuse me, i'm still pretty new at issuing pull requests.

My latest commit https://github.com/ansible/lightbulb/pull/139/commits/845cac70932594cf3d478f5f2c20cd37e60d8a72  contains the fixes to start the service before attempting to manipulate it.

Please let me know if it doesn't make sense.

Thanks

File: tools/aws_lab_setup/roles/common/tasks/RedHat.yml (hunk 1 above)
Comment (my0373) [inline tools/aws_lab_setup/roles/common/tasks/RedHat.yml]: Still getting the hang of github, but looks like my latest commit to my local branch automatically got added to the PR. Cool, but unexpected, hope it's not made things too messy.

**Comment by samdoran** [comment]
You did everything right.

With PRs, you push new commits to an existing branch, like you did, or you can amend your commit and force push if you want everything to be in one commit. It's not really necessary to use that second method anymore since GitHub now allows the project owner to do that when merging the branch.

Thanks for the contribution!

**Pull request merged**